scripts/docker/registry.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+
+services:
+  registry:
+    image: registry:2
+    container_name: rfaas-registry
+    ports:
+      - "5000:5000"
+    environment:
+      REGISTRY_AUTH: htpasswd
+      REGISTRY_AUTH_HTPASSWD_PATH: /auth/htpasswd
+      REGISTRY_AUTH_HTPASSWD_REALM: Registry Realm
+      REGISTRY_STORAGE_DELETE_ENABLED: "true"
+      #REGISTRY_HTTP_TLS_CERTIFICATE: /certs/domain.crt
+      #REGISTRY_HTTP_TLS_KEY: /certs/domain.unencrypted.key
+      #REGISTRY_HTTP_SECRET: supersecrettext
+    volumes:
+      - /home/ubuntu/rfaas/containers/registry:/var/lib/registry
```

Comment:
Are you referring to the volume mount?

Reply:
Yes, the mounting of `/home/ubuntu/rfaas/..`.

Reply:
Well, starting a registry does require a volume. The user can mount to wherever they please on their local machine. We should either (a) make it clear that this is a point of configuration, or (b) mount to a set place, like `/opt/rfaas/registry` or something like that.

Reply:
@mattnappo Then I think we need to make it clear that it needs to be adapted by the user. Can we maybe put some hardcoded value like `REGISTRY_LOCATION_REPLACE_ME`? Otherwise people might miss it easily.

Reply:
@mcopik Latest commit should resolve this. The documentation is still a WIP: I am going to add more to it.

I am also currently performing a final test of running the benchmarks with Docker. I will notify you when these tests pass, and then we can merge.